packages/app-staking/src/Overview/MyStake/index.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017-2020 @polkadot/app-staking authors & contributors
```

Comment:
The copyright should mention Centrality too.

Reply:
Sure. Will update it

Reply:
Updated

Reply:
Created a ticket in the backlog. IND-479